packages/main/src/plugin/provider-registry.ts

```diff
@@ -812,7 +815,10 @@ export class ProviderRegistry {
 
   getMatchingProviderLifecycleContextByProviderId(
     providerId: string,
-    providerConnectionInfo: ProviderContainerConnectionInfo | ProviderKubernetesConnectionInfo,
+    providerConnectionInfo:
+      | ProviderContainerConnectionInfo
```

Comment:
```suggestion
      ProviderContainerConnectionInfo
```
should we remove this first `|`